categories/categoriesRouter.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const categoriesRouter = express.Router();
+
+const Category = require('./categoriesModel');
+
+categoriesRouter.post('/new', (request, response) => {
+  const categoryInfo = request.body;
```

Comment:
You're very trusting that what's on the request.body will always be the data you want